container/import.bzl

```diff
@@ -40,9 +42,16 @@ load(
     _join_path = "join",
 )
 
-def _unzip_layer(ctx, zipped_layer):
-  unzipped_layer = _gunzip(ctx, zipped_layer)
-  return unzipped_layer, _sha256(ctx, unzipped_layer)
+def _layer_pair(ctx, layer):
+  zipped = False
+  for filetype in tgz_filetype:
+    if layer.basename.endswith(filetype):
+      zipped = True
+      break
+
+  zipped_layer = _gzip(ctx, layer) if not zipped else layer
```

Comment:
I think it reads slightly better as:
```python
zipped_layer = layer if zipped else _gzip_layer(ctx, layer)
unzipped_layer = layer if unzipped else _gunzip_layer(ctx, layer)
```

Even the second I think reads better as `not zipped`, since we've made the predicates match the sense of the target variable.

Total nit.

Reply:
Done.